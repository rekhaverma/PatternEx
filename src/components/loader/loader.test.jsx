import React from 'react';
import Loader from './loader.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<Loader />', () => {
  it('should match with snapshot', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
