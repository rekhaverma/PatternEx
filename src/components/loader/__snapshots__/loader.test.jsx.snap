// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should match with snapshot 1`] = `
<div
  className="loader"
>
  <div
    className="loader__content"
  >
    <div
      className="bullet"
    />
    <div
      className="bullet"
    />
    <div
      className="bullet"
    />
    <div
      className="bullet"
    />
    <div
      className="loader__text"
    >
      Crunching Numbers. Please wait!
    </div>
  </div>
</div>
`;
